GoogleSignIn/Sources/GIDSignInInternalOptions.h

```diff
@@ -91,6 +95,7 @@ NS_ASSUME_NONNULL_BEGIN
                                       loginHint:(nullable NSString *)loginHint
                                   addScopesFlow:(BOOL)addScopesFlow
                                          scopes:(nullable NSArray *)scopes
+                                          nonce:(nullable NSString *)nonce
```

Comment:
Same: let's add a method instead of changing existing API.

Reply:
see my response above :)